raiden/api/v1/resources.py

```diff
@@ -0,0 +1,113 @@
+from webargs.flaskparser import use_args, use_kwargs, parser
```

Comment:
It's added, but since it's the first line it does not appear as 'outdated' diff in github.